package.json

```diff
@@ -25,6 +25,12 @@
     "@lifterlms/dev": "^0.0.4-alpha.0",
     "@lifterlms/llms-e2e-test-utils": "^4.0.0",
     "@lifterlms/scripts": "^3.1.0",
+    "@wordpress/components": "^23.4.0",
+    "@wordpress/data": "^8.4.0",
+    "@wordpress/edit-post": "^7.5.0",
+    "@wordpress/element": "^5.4.0",
+    "@wordpress/i18n": "^4.27.0",
+    "@wordpress/plugins": "^5.5.0",
```

Comment:
these changes do not seem to be needed, are they?
I could run a build without these.

`@lifterlms/scripts` requires all the wordpress stuff.

Reply:
Yeah they aren't needed, these packages are always available in core.

The only reason to add them to package.json is to prevent IDE's throwing a warning that the package is not installed.

`@lifterlms/scripts` only requires the dev/webpack related packages (not in core)

Reply:
What do you mean?
I'm not actually sure "who" requires what but I'm sure that after running `npm install` then I have those installed in my `node_modules`, eheh.
So unless we need to force some version because the various deps require a different one, let's not change this, what do you think?

Reply:
When webpack compiles the scripts, it doesn't include core WP packages in the compiled JS. Instead, the dependencies are added to the asset PHP file and loaded separately - https://github.com/gocodebox/lifterlms-blocks/blob/7d40f4330a2497087019df8493efed09b809a99d/assets/js/llms-blocks.asset.php#L1

I'm not sure how they are being added to `node_modules`, maybe the lock file?

I have removed them here and happy either way, although, I think it might be best practise to include them in `package.json` so we can see which packages are being used?

Reply:
> When webpack compiles the scripts, it doesn't include core WP packages in the compiled JS.

Yup

> I'm not sure how they are being added to node_modules, maybe the lock file?

nope, removed both `node_modules` and `package-lock.json`

Ok if you think it's better ok.
But I think one of our dependencies it's requiring them.
I don't think we need to specify there _everything_ otherwise why don't specify babel ecc? It'll be massive.